index.js

```diff
@@ -816,12 +822,26 @@ function processIntakeErrorResponse (res, buf) {
   return err
 }
 
-function processConfigErrorResponse (res, buf) {
-  const err = new Error('Unexpected APM Server response when polling config')
+// Construct or decorate an Error instance from a failing response from the
```

Comment:
nit: Fine as is, but long term we should talk about whether we want to standardize on 

```
/**
 * One Line Description
 * ...
 * @param
 */
```

or 
```
// One Line Description
// ...
// @param 
```

Reply:
Indeed!  Obviously I've shown my hand here. :)